packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -154,7 +156,9 @@ const execute = (ipc, event, ids, args = []) => {
   }
 }
 
-module.exports = (ipc, pluginsFile) => {
+let registerTS = false
```

Comment:
Can we change this to `tsRegistered`? That makes it clear we're tracking whether or not it has already been registered. When I read this as it is, I thought it was meant to indicate whether or not we _will_ register typescript at all.

Reply:
Right. `tsRegistered` is much more cleared. Changed.